Scripts/Python/xHighLevelStarTrekDoor.py

```diff
@@ -49,16 +49,26 @@
 
 from Plasma import *
 from PlasmaTypes import *
+from enum import IntEnum
 
 
 strDoorClosedVar =  ptAttribString(1, "Door Closed SDL Var")
 xrgnDoorBlocker = ptAttribExcludeRegion(2,"Exclude Region")
 rgnSensor = ptAttribActivator(3, "Region Sensor")
-respOpenDoor = ptAttribResponder(4, "Open door Responder", netForce=0)
-respCloseDoor = ptAttribResponder(5, "Close door Responder", netForce=0)
+respOpenDoor = ptAttribResponder(4, "Open door Responder", netForce=False, netPropagate=False)
+respCloseDoor = ptAttribResponder(5, "Close door Responder", netForce=False, netPropagate=False)
 strDoorEnabledVar = ptAttribString(6, "Door Enabled SDL Var (optional)")
 
-doorSDLstates = {'closed':0,'opening':1,'open':2,'closing':3,'opentoclose':4,'closetoopen':5,'movingopen':6,'movingclosed':7}
+#doorSDLstates = {'closed':0,'opening':1,'open':2,'closing':3,'opentoclose':4,'closetoopen':5,'movingopen':6,'movingclosed':7}
+class doorSDLstates(IntEnum):
+    closed = 0
+    opening = 1
+    open = 2
+    closing = 3
+    opentoclose = 4
+    closetoopen = 5
+    movingopen = 6
+    movingclosed = 7
```

Comment:
I think I want us to work on these state names so that what is going on is a little more clear. I know that I still need to open this file in VS Code (instead of viewing the diff, which is basically useless) to really understand what all is going on here.